README.md

```diff
@@ -450,7 +450,7 @@ task_definition:
   task_execution_role: string            // Needed to use Cloudwatch Logs or ECR with your ECS tasks
   task_size:                             // Required for running tasks with Fargate launch type
     cpu_limit: string
-    mem_limit: string
+    mem_limit: string                    // Number-only values default to MiB
```

Comment:
I realize this is unrelated to your pull request, but could we make the comments in lines 454 and 455 line up with the others?